Lambdas/GHHooks/Models/+Document.swift

```diff
@@ -0,0 +1,10 @@
+import Markdown
+
+extension Document {
+    func filterOutChildren<T>(ofType: T.Type) -> Document where T: BlockMarkup {
+        let children = self.blockChildren.filter { markup in
+            type(of: markup) != ofType.self
+        }
+        return Document(children)
+    }
```

Comment:
```suggestion
    func filterOutChildren(ofType type: (some BlockMarkup).Type) -> Document {
        Document(self.blockChildren.filter { type(of: $0) != type }
    }
```
Might as well take advantage of all that fancy new stuff 5.7 enabled for generics 🙂.

Bonus chatter:
If I were writing for 5.6, or for any other reason couldn't simplify it with an opaque type, I'd write it like this:
```swift
    func filterOutChildren<T: BlockMarkup>(ofType _: T.Type) -> Document {
        Document(self.blockChildren.filter { type(of: $0) != T.self })
    }
```
According to the general rule that a non-opaque metatype parameter should always be referred to by its generic argument name, not the parameter's local name, whenever possible. (However, worth noting you _did_ correctly use an inequality comparison between the types. A common error in this kind of logic is to use a conformance check, e.g. `!(type(of: $0) is T.Type)`. (Apologies if the following is already known to you, I'm pontificating as usual 😆) The conformance test would happen to also have the desired result in this particular use case, but that's a syntactic quirk; a conformance test between metatypes checks "can type A be treated as type B" (which gets really fun when `Any.Type` gets involved), not "are type A and type B the same type".